.github/workflows/ci.yml

```
@@ -0,0 +1,275 @@
+      
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8.14.1
+          run_install: false
+      
+      - name: Get pnpm store directory
+        shell: bash
+        run: |
+          echo "STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
+
+      - uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)